test/e2e/40_tests.bats

```diff
@@ -70,7 +70,24 @@ function setup() {
   [ "$output" = 'False' ]
 }
 
-@test "When test.enable and rollback.enable are set, releases with failed tests are rolled back" {
+@test "When tests fail and test.ignoreFailures is true, release has phase 'TestFailed' and Released condition is True & Tested condition is False" {
```

Comment:
This test checks for ignored test failures on an install, it would be nice to have a test for ignored test failures on an upgrade as well. Essentially, would be a copy of this test with a successful install added at the beginning, or possibly add this code to the end of the existing "test success" test, if we want to avoid adding a new test for speed considerations.

Reply:
Or to save on execution time could just tack the upgrade test on to the end of this test, by making some small patch to the same HelmRelease which would trigger the upgrade with ignored failed tests. Would want to ensure that the upgrade does in fact occur (e.g. checking for upgrade log message), since the expected end condition state of the upgrade will be the same as the install.

Reply:
done